### PR TITLE
Update install.bat.example

### PR DESCRIPTION
**Pull request by PeterDaveHello**
cc c7b71c1ad17efa6fef70457857d0b91110db1356

### CODE REVIEW

File: script/install.bat.example [hunk 1]
```diff
@@ -31,7 +31,7 @@ exit /B
 
 setlocal & cd /d %~dp0
 
-set GIT_LFS_BIN_PATH="%LOCALAPPDATA%\GitMedia\bin"
+set GIT_LFS_BIN_PATH="%LOCALAPPDATA%\git-lfs\bin"
```
Comment (rubyist) [inline script/install.bat.example]: Let's make this GitLFS to remain consistent with how it was

Comment (PeterDaveHello) [inline script/install.bat.example]: Sure.

**Comment by PeterDaveHello** [comment]
-> #223

File: script/install.bat.example (hunk 1 above)
Comment (bumxnhoc) [inline script/install.bat.example]: Update at line note .

**Pull request closed without merging**